creating/export-blender-to-aframe/index.md

```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
 ## Finding a model on Sketchfab
 
 ## View the demo
+
+In this guide, we’ll cook up an A-Frame scene containing a model from SketchFab. I chose a model of a Rift CV1 by Ti Kawamoto. It’s a nice model created with Blender, that goes with the theme of WebVR. Plus, the thought of recursive “VR within VR” is amusing..
+For this lesson, I’m using Windows because that is what most VR devs use, but do please reach out to me if you have any questions about doing this in another OS! If you are comfortable setting up a site on a local dev server, feel free to ignore the parts about setting up http-server.
```

Comment:
yep, twitter.

I'm probably going to re-do this with browser-sync actually, since http-server doesn't seem to work to view locally on a phone! Yes, I will link to the repo; there's some useful stuff in the org, though: https://github.com/BrowserSync